README.md

```diff
@@ -416,3 +416,9 @@ export NUM_EXTRA_WORKERS=2
 
 After initial deployment, a file containing the BareMetalHost manifests can be
 applied, e.g `oc apply -f ocp/ostest/extra_host_manifests.yaml`
+
+After that, scale up the machineset to provision the extra workers:
+
+```
+oc scale machineset ostest-578sb-worker-0 --replicas=4 -n openshift-machine-api
```

Comment:
I think we should either add the output from `oc get machineset` here to be complete clear where the name comes, or else make the ostest-578sb-worker-0 a more obvious placeholder?